nautobot/project-static/css/dark.css

```diff
@@ -6,38 +6,24 @@ html[data-theme="dark"] {
   filter: invert(1) hue-rotate(180deg);
 }
 /* Ignores inverting images for dark theme */
-html[data-theme="dark"] img {
-  filter: invert(1) hue-rotate(180deg);
-}
 /* Makes badge text white in dark mode */
-html[data-theme="dark"] .badge {
-  color: black;
-}
 /* White text on buttons */
-html[data-theme="dark"] button {
-  color: black;
-}
 /* White text on buttons */
-html[data-theme="dark"] .btn {
-  color: black;
-}
 /* White text on labels */
-html[data-theme="dark"] .label {
-  color: black !important;
-}
 /* Colors for labels like with device role, status, etc */
-html[data-theme="dark"] .color-block {
-  filter: invert(1) hue-rotate(180deg);
-}
+/* Selected choice in a select2 widget (rendered as a tag) */
 /* Dropdown list when selecting a color for status, device role, etc */
-html[data-theme="dark"] .select2-results__option {
-  filter: invert(1) hue-rotate(180deg);
-  color: #ccc !important;
-}
 /* Dropdown list for showing selected color for status, device role, etc */
```

Comment:
Can we consolidate all of these various comments that you've left here, and make sure they're all still accurate? I'd think something like simply:

```css
/* Double-invert the following elements, restoring them to the original non-inverted colors */
```